service/src/main/resources/application.yaml

```diff
@@ -133,6 +135,8 @@ fpl:
     mapping: 'SA=>Cafcass Cymru:FamilyPublicLaw+cafcassWales@gmail.com;HN=>Cafcass:FamilyPublicLaw+cafcassEngland@gmail.com'
   local_authority_code_to_shared_inbox:
     mapping: 'SA=>FamilyPublicLaw+sa@gmail.com;HN=>FamilyPublicLaw+hn@gmail.com'
+  default_email:
+    mapping: 'FamilyPublicLaw+PublicLawEmail@gmail.com'
```

Comment:
going by the AC's, should the email be `PublicLawAndAdoption@justice.gov.uk` or `FamilyPublicLaw+PublicLawEmail@gmail.com`?

Reply:
Yes we are using `FamilyPublicLaw+PublicLawEmail@gmail.com` for tests and `PublicLawAndAdoption@justice.gov.uk` will be mapped in Production az vault.